example/provision-aks-via-resource/tfvars/demo.tfvars

```diff
@@ -0,0 +1,35 @@
+# Infra variables
```

Comment:
should we be committing this?

Reply:
The file is meant to show an example of what needs to be filled out.